<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameOfLife</class>
 <widget class="QMainWindow" name="GameOfLife">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GameOfLife</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>451</width>
      <height>259</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="mainLayout_5">
     <item>
      <layout class="QVBoxLayout" name="gameLayout_5"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="setLayout_5">
       <item>
        <layout class="QHBoxLayout" name="controlLayout_5">
         <item>
          <widget class="QPushButton" name="startButton_5">
           <property name="text">
            <string>Start</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_6">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="stopButton_5">
         <property name="text">
          <string>Stop</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_7">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="clearButton_5">
         <property name="text">
          <string>Wyczyść</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Rozmiar planszy</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="cellsControl_5">
         <property name="suffix">
          <string> cells</string>
         </property>
         <property name="minimum">
          <number>10</number>
         </property>
         <property name="maximum">
          <number>400</number>
         </property>
         <property name="value">
          <number>50</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label2_5">
         <property name="text">
          <string>Częstotliwość</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="iterInterval_5">
         <property name="suffix">
          <string> ms</string>
         </property>
         <property name="minimum">
          <number>50</number>
         </property>
         <property name="maximum">
          <number>100000</number>
         </property>
         <property name="singleStep">
          <number>10</number>
         </property>
         <property name="value">
          <number>100</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLCDNumber" name="howManySteps"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="fileLayout_5">
         <item>
          <widget class="QPushButton" name="loadButton_5">
           <property name="text">
            <string>Wczytaj plik</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="saveButton_5">
           <property name="text">
            <string>Zapisz</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_5">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
